src/components/AllMovies/AllMovies.js

```diff
@@ -17,10 +18,16 @@ const AllMovies = ({ movies, handleClick, goHome }) => {
   })
 
   return (
-    <div className='card-container card-carousel'>
+    <div className='card-container'>
       {allMovies}
     </div>
   )
 }
 
-export default AllMovies
\ No newline at end of file
+export default AllMovies
+
+AllMovies.propTypes = {
+  movies: PropTypes.object,
```

Comment:
To really get the benefit of proptypes, could you go more in-depth here so that it is typechecking not just for an object, but that the contents of that object are what you'd expect?